pybboxes/types/albumentations_bounding_box.py

```diff
@@ -34,6 +34,35 @@ def to_voc(self, return_values: bool = False) -> Union[Tuple[int, int, int, int]
             return x_tl, y_tl, x_br, y_br
         return BoundingBox(x_tl, y_tl, x_br, y_br, image_size=self.image_size, strict=self.strict)
 
+    def shift(self, amount: Tuple[float, float]) -> "AlbumentationsBoundingBox":
+        """Returns a new bounding box shifted by the given thresholds. The new
+        bounding box has same image shape, and other properties as the current
+        object.
+
+        Parameters
+        ----------
+        amount: Tuple[float, float]
+            The amount to shift the bounding box. The first value is the
+                amount to shift the x-coordinate, and the second value is the
+                amount to shift the y-coordinate.
+
+        Returns
+        -------
+        AlbumentationsBoundingBox
+            The new bounding box.
+        """
+        horizontal_threshold, vertical_threshold = amount
```

Comment:
Can we rename these also (take this comment into consideration for all box types please)

`horizontal_threshold, vertical_threshold` -> `horizontal_shift, vertical_shift`

Reply:
Done :)